bin/check-es-indices-sizes.rb

```diff
@@ -161,9 +161,9 @@ def run
     nodes_being_used = node_fs_stats['nodes'].values.select { |node| node['indices']['store']['size_in_bytes'] > 0 }
     # TODO: come back and cleanup all these rubocop disables with a little refactor
     # rubocop:disable Style/SingleLineBlockParams
-    used_in_bytes = nodes_being_used.map { |node| node['fs']['data'].map { |data| data['total_in_bytes'] - data['available_in_bytes'] }.flatten }.flatten.inject { |sum, x| sum + x } # rubocop:disable LineLength
+    # rubocop:disable LineLength
```

Comment:
if you are not disabling rubocop inline you should then enable it when you are done.

Reply:
rubocop is complaining about line being too long here, hence the disable directive.

Reply:
`# rubocop:disable Metrics/LineLength` on the same line.